sonar-core/src/main/java/org/sonar/core/issue/DefaultIssue.java

```diff
@@ -267,34 +267,49 @@ public DefaultIssue setLine(@Nullable Integer l) {
     return this;
   }
 
+  @Deprecated
   @Override
   @CheckForNull
   public Double effortToFix() {
-    return effortToFix;
+    return gap();
   }
 
-  public DefaultIssue setEffortToFix(@Nullable Double d) {
-    Preconditions.checkArgument(d == null || d >= 0, format("Effort to fix must be greater than or equal 0 (got %s)", d));
-    this.effortToFix = d;
+  @Override
+  @CheckForNull
+  public Double gap() {
+    return gap;
+  }
+
+  public DefaultIssue setGap(@Nullable Double d) {
+    Preconditions.checkArgument(d == null || d >= 0, format("Gap must be greater than or equal 0 (got %s)", d));
+    this.gap = d;
     return this;
   }
 
+  @Deprecated
+  @Override
+  @CheckForNull
+  public Duration debt() {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the missing @deprecated Javadoc tag. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)